Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1594,57 +1579,62 @@ namespace AzToolsFramework
+            }
+            else
+            {
+                // note: must use sorted entityIds based on hierarchy order when updating transforms
+                for (AZ::EntityId entityId : sharedScaleState->m_entityIds)
                 {
-                    return vec.GetX() + vec.GetY() + vec.GetZ();
-                };
+                    auto entityIdLookupIt = m_entityIdManipulators.m_lookups.find(entityId);
+                    if (entityIdLookupIt == m_entityIdManipulators.m_lookups.end())
+                    {
+                        continue;
+                    }
 
-                const float uniformScale =
-                    action.m_start.m_sign * sumVectorElements(action.LocalScaleOffset() - sharedScaleState->m_savedScaleOffset);
-                const float scale = AZ::GetClamp(1.0f + uniformScale / initialScale, AZ::MinTransformScale, AZ::MaxTransformScale);
-                const AZ::Transform scaleTransform = AZ::Transform::CreateUniformScale(scale);
+                    const AZ::Transform initial = entityIdLookupIt->second.m_initial;
+                    const float initialScale = initial.GetUniformScale();
 
-                switch (InfluenceFromModifiers(action.m_modifiers))
-                {
-                case Influence::Individual:
-                    {
-                        const AZ::Transform pivotTransform = TransformNormalizedScale(entityIdLookupIt->second.m_initial);
-                        const AZ::Transform transformInPivotSpace = pivotTransform.GetInverse() * initial;
+                    const float scale = AZ::GetClamp(1.0f + uniformScale / initialScale, AZ::MinTransformScale, AZ::MaxTransformScale);
+                    const AZ::Transform scaleTransform = AZ::Transform::CreateUniformScale(scale);
 
-                        SetEntityWorldTransform(entityId, pivotTransform * scaleTransform * transformInPivotSpace);
-                    }
-                    break;
-                case Influence::Group:
+                    switch (InfluenceFromModifiers(action.m_modifiers))
                     {
-                        const AZ::Transform pivotTransform =
-                            TransformNormalizedScale(m_entityIdManipulators.m_manipulators->GetLocalTransform());
-                        const AZ::Transform transformInPivotSpace = pivotTransform.GetInverse() * initial;
+                    case Influence::Individual:
+                        {
+                            const AZ::Transform pivotTransform = TransformNormalizedScale(entityIdLookupIt->second.m_initial);
+                            const AZ::Transform transformInPivotSpace = pivotTransform.GetInverse() * initial;
 
-                        SetEntityWorldTransform(entityId, pivotTransform * scaleTransform * transformInPivotSpace);
+                            SetEntityWorldTransform(entityId, pivotTransform * scaleTransform * transformInPivotSpace);
```

Comment:
Not necessarily a job for this PR, but I think this logic can get simplified since the scale transform is uniform now.

Reply:
That's a good call, I'll take another quick look! Likely worth doing now as we're here 🙂

Reply:
Hmm as this just changed due to indentation I'd rather not get into modifying this now